TestFoundation/TestFileManager.swift

```diff
@@ -8,7 +8,7 @@
 //
 
 #if NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT
-    #if (os(Linux) || os(Android))
+    #if os(Linux) || os(Android) || os(Windows)
         @testable import Foundation
     #else
         @testable import SwiftFoundation
```

Comment:
Would it be better to use:
```
#if canImport(SwiftFoundation)
  @testable import SwiftFoundation
#else
  @testable import Foundation
#endif
```

Rather than having an ever growing list of OSes or would this not work?